src/20230925_AaveV3_Eth_ExpansionOfOrbit/AaveV3_Ethereum_ExpansionOfOrbit_20230925.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.16;
+
+// testing libraries
+import 'forge-std/Test.sol';
+
+// contract dependencies
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV2Ethereum, ICollector} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveMisc, IStreamable} from 'aave-address-book/AaveMisc.sol';
+import {AaveV3_Ethereum_ExpansionOfOrbit_20230925} from './AaveV3_Ethereum_ExpansionOfOrbit_20230925.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+
+// make test-contract filter=AaveV3_Ethereum_ExpansionOfOrbit_20230925
+
+contract AaveV3_Ethereum_ExpansionOfOrbit_20230925Test is Test {
+  IERC20 public constant GHO = IERC20(AaveV3EthereumAssets.GHO_UNDERLYING);
+
+  // 0x464
+  ICollector public immutable AAVE_COLLECTOR = AaveV2Ethereum.COLLECTOR;
+  address public constant STABLE_LABS = 0x9c489E4efba90A67299C1097a8628e233C33BB7B;
```

Comment:
Better to make test of all participants, just do a function and call it in a loop

Reply:
fixed